inyoka_theme_default/locale/de_CH/LC_MESSAGES/django.po

```diff
@@ -659,6 +435,15 @@ msgstr ""
 msgid "Browse pastes"
 msgstr ""
 
+msgid ""
+"But the article itself is not listed as an attachment, perhaps\n"
+"          these pages are using the article mistakenly or an old revision\n"
```

Comment:
Jepp. Either we can remove the line breaks or just wait until https://github.com/pallets/jinja/pull/683 will be available upstream. ^^